src/modules/users/ActiveDirectoryJob.cpp

```diff
@@ -0,0 +1,91 @@
+/* === This file is part of Calamares - <https://calamares.io> ===
+ *
+ *   SPDX-FileCopyrightText: 2024 Simon Quigley <tsimonq2@ubuntu.com>
+ *   SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+#include "ActiveDirectoryJob.h"
+
+#include "Config.h"
+
+#include "GlobalStorage.h"
+#include "JobQueue.h"
+#include "utils/Logger.h"
+#include "utils/Permissions.h"
+#include "utils/System.h"
+
+#include <QDateTime>
+#include <QDir>
+#include <QFile>
+#include <QFileInfo>
+#include <QTextStream>
+#include <QProcess>
+
+ActiveDirectoryJob::ActiveDirectoryJob(QStringList& activeDirectoryInfo)
+    : Calamares::Job()
+    , m_activeDirectoryInfo(activeDirectoryInfo)
+{
+}
+
+QString
+ActiveDirectoryJob::prettyName() const
+{
+    return tr( "Enroll system in Active Directory" );
```

Comment:
These strings typically get an annotation (`@title` for `prettyName()`, for instance). Take a look at the way translations are done in other modules. This helps the translators with a little context to show how to translate otherwise-identical-in-English strings. You can ask @bitigchi (also in KDE-translation channels, I think) for hints.